python/social_bot/envs/grocery_ground_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
```

Comment:
Did you follow CONTRIBUTING.md step 2. It should add more stuff to the head?

Reply:
Added. 
The file was modified from a copy of simple_navigation_test.py, pre-commit won't add more license info if there is already a simple one. Also add changed simple_navigation_test.py